src/index.ts

```diff
@@ -1,5 +1,6 @@
-import { AxiosInstance, AxiosRequestConfig } from 'axios'
-import jwtDecode from 'jwt-decode'
+import axios, { AxiosInstance, AxiosRequestConfig } from 'axios'
+import jwtDecode, { JwtPayload } from 'jwt-decode'
+import Storage from './storage'
```

Comment:
Be careful, I am not sure if the developer is using React Native with expo, the compiler is going to understand which file it should execute.

I don't know if differentiating the files with `.android` and `.ios` works with expo. I always used `Platform`, but I don't have much experience with react native tbh.

Also, I am not sure if you want to take that into account

Reply:
React Native will look for platform specific extensions first as mentioned in the react native docs:
https://reactnative.dev/docs/platform-specific-code#platform-specific-extensions